NuGet.Config

```diff
@@ -42,7 +46,8 @@
       <package pattern="xunitxml.testlogger" />
     </packageSource>
     <packageSource key = "dotnet-eng">
-      <package pattern="microsoft.*" />
+      <package pattern="Microsoft.DotNet.*" />
+      <package pattern="Microsoft.DotNet.Build.Tasks.Feed " />
```

Comment:
the second pattern, the package specific pattern, isn't needed here if the same package ID isn't mapped to another feed as well.